### PR TITLE
Updated GitOps RN 1.10.0 with a missed deprecated issue.

### PR DESCRIPTION
**Pull request by LunguTheEngineer**
**This  PR addreses a customer issue missed during the release of GitOps 1.10.0**

- Aligned team: DevTools
- GitOps version: V1.10.0

- JIRA issue: [RHDEVDOCS-5664](https://issues.redhat.com/browse/RHDEVDOCS-5664)
- Preview pages: [GitOps 1.10.0 RNs](https://66270--docspreview.netlify.app/openshift-gitops/latest/release_notes/gitops-release-notes.html#deprecated-features-1-10-0_gitops-release-notes)
- SME review: @reginapizza , @svghadi 
- QE review: @trdoyle81
- Peer review: @ekristova , @Srivaralakshmi @stevsmit

### CODE REVIEW

**Comment by Srivaralakshmi** [comment]
@masauso-lungu This is the preview link: https://66270--docspreview.netlify.app/openshift-gitops/latest/release_notes/gitops-release-notes.html

**Comment by svghadi** [comment]
/lgtm
Thanks @masauso-lungu

**Comment by masauso-lungu** [comment]
Thank you for the preview link @Srivaralakshmi

**Review by svghadi — APPROVED** [review]

File: modules/gitops-release-notes-1-10-0.adoc [hunk 1]
```diff
@@ -102,6 +102,8 @@ As an alternative to `.spec.resourceCustomizations`, you can use `.spec.resource
 To prevent data loss during upgrade to {gitops-title} Operator v1.10.0, ensure that you backup `.spec.resourceCustomization` value if it is used in your Argo CD CRs.
 ==== 
 
+* With this update, support for deprecated Legacy Configuration Management Plugins (CMPs) specified in `argo-cm` or through the Operator `argocd.spec.configManagementPlugins` is removed in Argo CD v2.8. To continue using your legacy plugins, consider migrating them to the new sidecar available in the Operator through `argocd.spec.repo.sidecarContainers`. link:https://issues.redhat.com/browse/GITOPS-3462[GITOPS-3462]
```
Comment (ekristova) [inline modules/gitops-release-notes-1-10-0.adoc:105]: I am not sure that the word "Operator" is accurate here as the `configManagementPlugins` object is described as Argo CD cm property here in this table: [Argo CD custom resource properties](https://docs.openshift.com/gitops/1.10/argocd_instance/argo-cd-cr-component-properties.html#argo-cd-properties_argo-cd-cr-component-properties). But I am not sure. Could you please check?

```suggestion
* With this update, support for deprecated Legacy Configuration Management Plugins (CMPs) specified in `argo-cm` or through the `argocd.spec.configManagementPlugins` parameter is removed in Argo CD v2.8. To continue using your legacy plugins, consider migrating them to the new sidecar available in the Operator through `argocd.spec.repo.sidecarContainers`. link:https://issues.redhat.com/browse/GITOPS-3462[GITOPS-3462]
```

Comment (ekristova) [inline modules/gitops-release-notes-1-10-0.adoc:105]: ... support for deprecated Legacy Configuration Management Plugins (CMPs) specified in  the `argo-cm` config map **or** through the Operator `argocd.spec.configManagementPlugins` is removed ...

The "or" here seems ambiguous. Does it mean "either, or"  or can we choose where it is specified, or is it specified in both places? Please check and make changes accordingly.

Comment (masauso-lungu) [reply modules/gitops-release-notes-1-10-0.adoc:105]: Hello @svghadi , would you please assist in clarifying this ?

Comment (masauso-lungu) [reply modules/gitops-release-notes-1-10-0.adoc:105]: @svghadi here too, need clarity

Comment (svghadi) [reply modules/gitops-release-notes-1-10-0.adoc:105]: It's **"either, or"**

Comment (svghadi) [reply modules/gitops-release-notes-1-10-0.adoc:105]: The "**\<type-of-object\>**" here is "**field in Argo CD CR**".

[the remark this answers is not in the document]
Comment (svghadi) [reply modules/gitops-release-notes-1-10-0.adoc:105]: "**\<type-of-object\>**" = "**field in Argo CD CR**".

Comment (masauso-lungu) [reply modules/gitops-release-notes-1-10-0.adoc:105]: @ekristova , I have addressed this now.

[the remark this answers is not in the document]
Comment (masauso-lungu) [reply modules/gitops-release-notes-1-10-0.adoc:105]: Addressed!, thank you @svghadi

File: modules/gitops-release-notes-1-10-0.adoc [hunk 2]
```diff
@@ -102,6 +102,8 @@ As an alternative to `.spec.resourceCustomizations`, you can use `.spec.resource
 To prevent data loss during upgrade to {gitops-title} Operator v1.10.0, ensure that you backup `.spec.resourceCustomization` value if it is used in your Argo CD CRs.
 ==== 
 
+* With this update, support for deprecated Legacy Configuration Management Plugins (CMPs) specified in the `argocd-cm` config map or in the Operator through the `.spec.configManagementPlugins` field in Argo CD CR is removed in Argo CD v2.8. To continue using your legacy plugins, consider migrating them to the new sidecar available in the Operator through the `.spec.repo.sidecarContainers` field in Argo CD CR. link:https://issues.redhat.com/browse/GITOPS-3462[GITOPS-3462]
```
Comment (Srivaralakshmi) [inline modules/gitops-release-notes-1-10-0.adoc:105]:
```suggestion
* With this update, support for deprecated legacy Configuration Management Plugins (CMPs) specified in the `argocd-cm` config map or the Operator through the `.spec.configManagementPlugins` field in Argo CD CR is removed in Argo CD v2.8. To continue using your legacy plugins, consider migrating them to the new sidecar available in the Operator through the `.spec.repo.sidecarContainers` field in Argo CD CR. link:https://issues.redhat.com/browse/GITOPS-3462[GITOPS-3462]
```

[the remark this answers is not in the document]
Comment (masauso-lungu) [reply modules/gitops-release-notes-1-10-0.adoc:105]: You are right @Srivaralakshmi , it should be lower case.

**Comment by stevsmit** [comment]
@masauso-lungu @Srivaralakshmi Hiya, 

PRs need to go through the peer review process by a member of the OCP docs team peer review squad. You can do this by adding the `peer-review-needed` label prior to adding the `merge-review-needed` label. This is for future reference.

**Comment by masauso-lungu** [comment]
Hello @stevsmit, but since peer review has already been conducted by @Srivaralakshmi and @ekristova ,wouldn't a `peer-review-done` label surfice  at this moment?

**Comment by stevsmit** [comment]
From what I understand on the OCP docs side of things, Sri and Eliska aren't part of the OCP docs peer review squad (for a full list, see https://docs.google.com/spreadsheets/d/1EV8c9cPYK1HymBtiIQZX2piyuYKp0rurZ7afHPkBXWo/edit#gid=0). I know that Sri is going to be onboarded soon. This is just our process.

**Pull request merged**